### PR TITLE
fix: erase resources folder content before pulling resources

### PR DESCRIPTION
**Pull request by y-lakhdar**
https://coveord.atlassian.net/browse/CDX-695

## Proposed changes

Simply reset the resources/ folder before doing another `org:resources:pull`.
 
## Testing

- [ ] Unit Tests:

- [x] E2E Tests:

- [x] Manual Tests:

### CODE REVIEW

File: packages/cli/src/commands/org/resources/pull.ts [hunk 1]
```diff
@@ -59,6 +59,11 @@ export default class Pull extends Command {
       default: true,
       allowNo: true,
     }),
+    override: flags.boolean({
```
Comment (louis-bompart) [inline packages/cli/src/commands/org/resources/pull.ts:62]: I think you're misusing the word override here.
It should be overwrite I think :)

File: packages/cli/src/commands/org/resources/pull.ts [hunk 2]
```diff
@@ -115,6 +122,21 @@ export default class Pull extends Command {
     await project.refresh(snapshotBlob);
   }
 
+  private async ensureProjectReset(project: Project) {
+    const {flags} = this.parse(Pull);
+    if (!flags.override && project.contains(Project.resourceFolderName)) {
+      const override =
+        await cli.confirm(dedent`There is already a Coveo project with resources in it.
+        This command will override the ${Project.resourceFolderName} folder content, do you want to proceed? (y/n)`);
```
Comment (louis-bompart) [inline packages/cli/src/commands/org/resources/pull.ts:130]: Same here, overwrite I think?

**Review by louis-bompart — APPROVED** [review]

**Review by louis-bompart — COMMENTED** [review]
Not sure about the use of `override` in the wording.

**Review by olamothe — APPROVED** [review]

File: packages/cli/src/commands/org/resources/pull.ts (hunk 1 above)
Comment (olamothe) [reply packages/cli/src/commands/org/resources/pull.ts:62]: Yep !

Comment (y-lakhdar) [reply packages/cli/src/commands/org/resources/pull.ts:62]: Oups. thanks

**Pull request merged**